src/lib/integration_test_lib/wait_condition.ml

```diff
@@ -202,11 +201,19 @@ struct
     let description =
       sprintf "[%d] snarked_ledgers to be generated since genesis" num_proofs
     in
+    let slots_for_first_proof =
+      Test_config.(
+        slots_for_blocks @@ blocks_for_first_ledger_proof test_config)
+    in
+    let slots_for_additional_proofs =
+      Test_config.slots_for_blocks (num_proofs - 1)
+    in
+    let total_slots = slots_for_first_proof + slots_for_additional_proofs in
     { id = Ledger_proofs_emitted_since_genesis
     ; description
     ; predicate = Network_state_predicate (check (), check)
-    ; soft_timeout
-    ; hard_timeout
+    ; soft_timeout = Network_time_span.Slots (total_slots + 6)
+    ; hard_timeout = Network_time_span.Slots (total_slots + 12)
```

Comment:
why is it `total_slots+6` and `total_slots+12`?  shouldn't it be based on a multiplier, like `(total_slots * 1.5)` and `(total_slots *2)`?